### DISCUSSION TITLE
Artisan 2.10.0 - Floating LCDs

### DISCUSSION THREAD

**Discussion by CarlosBabo** [artisan-roaster-scope/artisan#1338] [Versions]
Hi all.
Please Help.
After upgrading to the latest ver (2.10.0) my Floating LCDs (Main LCDs and Delta LCDs) disappear (Go to Background) if I press anywhere within the application.

Any support would be great.

**Comment by MAKOMO** [comment]
On which platform?

**Comment by DennieSch** [comment]
Hey Marko - I noticed the same thing. If you have Artisan at 'full screen' and turn on an LCD (View/xxxLCDs) it comes up on top of the main app window but, if you click on any part of the main app window the LCD goes behind the main app window. If you turn the LCD off (uncheck) and turn is back on it does come up on top of the main app window but goes behind again if you click on the app window. 
In my last 'roast session' I had to resized the app window to allow room to show/see the LCDs (while they were 'behind' the main app). 
Artisan - 2.10.0 (c44a180)
Platform - Windows 11, Version 22H2 on a Surface-Pro-8

Cheers,
Dennie

**Reply by CarlosBabo** [reply]
Hello. Did your issue get fixed?

**Reply by DennieSch** [reply]
Issue appears to be resolved on my setup - 
Artisan - 2.10.1 (46ed365) (Continuous Build)
Windows 11 Home, Version	23H2, OS build	22631.2861
Surface-Pro-8

**Reply by MAKOMO** [reply]
Thanks for reporting!

**Comment by CarlosBabo** [comment]
Hi there,
Using windows 11.

Artisan v2.8, works fine
